library/manageiq.py

```diff
@@ -152,28 +178,31 @@ def update_provider(self, provider_id, provider_name, endpoints):
         """ Updates the existing provider with new parameters
         """
         try:
+            update_time = time.strftime(ManageIQ.pattern, time.gmtime())
```

Comment:
Yeah, gmtime returns UTC.  (Unless the computer is misconfigured.)
Would love to somehow check this in tests this but can't think of a meaningful way.
BTW, I'm surprised you don't mock gmtime in tests.  Is update_provider scenario uncovered?

I'd pass around struct_time objects instead of formatting then parsing, but LGTM.